modules/iform_licences/iform_licences.module

```diff
@@ -68,13 +67,22 @@ function iform_licences_form_alter(&$form, &$form_state, $form_id) {
             );
           }
           else {
-            $description .= strtoupper(
+            $description .= '<b>' . strtoupper(
```

Comment:
Isn't \<strong\> a more up to date element than \<b\>?

Reply:
Updated.